doc/UsersGuide/source/FAQ.rst

```diff
@@ -150,10 +150,314 @@ places, and depends on whether you are using the write component to generate you
 ==============================================================
 How do I set the total number of tasks for my job?
 ==============================================================
-The total number of MPI tasks used by the UFS Weather Model is a combination of compute and quilt tasks, and can be calculated using the following relationship:
 
-   * total tasks = ``compute tasks + quilt tasks``
-   * compute tasks = ``x layout * y layout * number of tiles``
-   * quilt tasks = ``write_groups * write_tasks_per_group`` if ``quilting==.true.``
+In the UFS WM, each component's MPI task information, including the
+starting task and end ending task numbers and the number of threads,
+are specified using the component specific ``petlist_bounds`` and
+``omp_num_threads`` in the ``nems.configure``. In general, the total
+number of tasks required is the sum of all the component tasks, as
+long as those components do not overlap (i.e., share the same
+PETs). An example of a global 5 component coupled configuration
+nems.configure at the end of this section.
+
+FV3
+^^^
+FV3ATM consists of one or more forecast grid components and a write
+grid component.
+
+The MPI tasks for the forecast grid components are specified in the
+layout variable in one or more namelist files ``input*.nml``
+(e.g. input.nml and input_nest02.nml). The total number of mpi
+tasks is the product of the specified layout, summed over all
+domains. For example, for a global domain with 6 tiles and ``layout
+= 6,8``, the total number of forecast tasks is ``6*6*8 = 288``. For
+two regional domains using ``input.nml`` and ``input_nest02.nml``,
+each with ``layout = 6,10``, the total number of forecast tasks is
+``6*10 + 6*10 = 120``.
+
+FV3ATM will utilize the write grid component if ``quilting`` is set
+to .true. In this case, the mpi tasks of the write grid components is
+the product of the ``write_groups`` and the ``write_tasks_per_group``
+in the ``model_configure`` file.
+
+::
+
+   quilting:                .true.
+   write_groups:            1
+   write_tasks_per_group:   60
+
+
+The total number of write tasks is 60.
+
+The total number of MPI tasks for FV3ATM is the sum of the forecast
+MPI tasks and any write grid component tasks.
+
+::
+
+   total tasks = number of forecast tasks + number of write grid component tasks
+
+If ESMF-managed threading is used, the total number of tasks for
+the atmosphere component is given by the product of the number of
+threads requested and the MPI tasks (both forecast and write grid
+component). Since the ATM component is listed first in the
+``nems.configure``, the ATM tasks are given by
+
+::
+
+   ATM_petlist_bounds     0 total_tasks*number_of_threads-1
+   ATM_omp_num_threads    number_of_threads
```

Comment:
Maybe it will be clearer if we use number_of_threads_amt? I see we have "number_of_threads" for atm/wave/cmeps, we may let user know what the number_of_threads could be different for each component.